### PR TITLE
Array initialization and Aggregation function tests

### PR DESCRIPTION
**Pull request by faze-geek**

### CODE REVIEW

File: numpy/tests/test_numpy.py [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+    assert arr.mean(0) == np.array([[1.5, 2.5, 3.5], [3.5, 4.5, 5.5], [5.5, 6.5, 7.5]])
+    assert arr.mean(1) == np.array([[3.5, 4.5, 5.5]])
+    assert arr.mean(2) == np.array([[2.5, 4.5, 6.5]])
+
+def test_array_std():
+    arr = np.array([[[1.5, 2.5, 3.5], [3.5, 4.5, 5.5], [5.5, 6.5, 7.5]]])
+    assert arr.std() == 1.8257418583505538
+    assert arr.std(0) == np.array([[0.0, 0.0, 0.0], [0.0, 0.0, 0.0], [0.0, 0.0, 0.0]])
+    assert repr(arr.std(1)) == repr(np.array([[1.632993,  1.632993,  1.632993]]))
+    assert repr(arr.std(2)) == repr(np.array([[ 0.816497,  0.816497,  0.816497]]))
```
Comment (faze-geek) [inline numpy/tests/test_numpy.py:139]: @16bit-ykiko Do you have something better in mind here ? I guess due to some truncation, values of high floating point precision give false on equation (but obviously print the same).

Comment (16bit-ykiko) [reply numpy/tests/test_numpy.py:139]: Have you implemented `numpy.allclose`? I think you can use it to specifie tolerance.

Comment (faze-geek) [reply numpy/tests/test_numpy.py:139]: I was thinking of having that. Thanks for the validation.

**Comment by faze-geek** [comment]
Added `np.allclose` in the latest commit and tests working as expected. Merging.

**Pull request merged**